src/Microsoft.AspNetCore.Authentication.Cookies/CookieAuthenticationHandler.cs

```diff
@@ -165,7 +165,7 @@ private CookieOptions BuildCookieOptions()
             }
 
             // REVIEW: Should this check if there was an error, and then if that error was already handled??
```

Comment:
While we are here, we should resolve this really old REVIEW of mine :)  I think for cookies its safe to just ignore any errors and treat them equivalent to a null ticket, so we can probably just remove this review now.  @Tratcher agreed?